ConcurrentPercolator/ConcurrentPercolator.tla

```diff
@@ -185,9 +185,12 @@ canLockKey(k, ts) ==
     /\ writes = {}       \* no any newer writes or rollbacks.
 
 \* Locks the key and places the data.
-lockKey(k, start_ts, primary) ==
+lockKey(c, k, start_ts, primary) ==
   /\ key_lock' = [key_lock EXCEPT ![k] = @ \union {[ts |-> start_ts, primary |-> primary]}]
   /\ key_data' = [key_data EXCEPT ![k] = @ \union {[ts |-> start_ts]}]
```

Comment:
L48 states that `key_last_read_ts` is actually an auxiliary variables, but looks like now it should exist in real world implementation, can we fix that comment?

Reply:
No, in the real world, the `last_read_ts` only exists in the region level, not key level.

Reply:
By saying something is auxiliary, I mean it corresponds to nothing in a real-world implementation. Hereby, `last_read_ts` does have a mapping to **something** in implementation, though it is a little different. But remember TLA+ spec is something abstract, we don't have a region concept here, so saying  `last_read_ts` exists in real-world implementation is okay, IMHO.